src/authorization/event.hpp

```diff
@@ -0,0 +1,31 @@
+#include "cocaine/api/authorization/event.hpp"
+
+namespace cocaine {
+namespace authorization {
+namespace event {
+
+class disabled_t : public api::authorization::event_t {
```

Comment:
I don't like 'disabled' name, it' not disabled, it just allows everything for everybody